### PR TITLE
fix header emitted from gpad writer to match 2.0 specifications

### PR DESCRIPTION
**Pull request by sierra-moxon**
Note: I did not update the header in the tests that specifically request GPAD 1.2.  This PR should not be merged yet.

### CODE REVIEW

**Comment by sierra-moxon** [comment]
@dustine32 - can you take a look at the changes I made to validate.py here?  Not ready to merge yet, just wanted a second pair of eyes on the changes I made to carry forward the model details from noctua GPADs through to the final GPADs produced by the pipeline. :)

File: bin/validate.py [hunk 1]
```diff
@@ -329,6 +337,46 @@ def make_products(dataset, target_dir, gaf_path, products, ontology_graph):
...
+    print("noctua-gpad-file: {}".format(noctua_gpad_file))
+    if noctua_gpad_file is not None:
+        with open(noctua_gpad_file) as nf:
+            gpadparser = GpadParser(config=assocparser.AssocParserConfig(
+                ontology=ontology_graph,
+                paint=False,
+            ))
+
+            click.echo("Making noctua gpad products...")
+            with click.progressbar(iterable=gpadparser.association_generator(file=nf), length=lines) as associations:
```
Comment (dustine32) [inline bin/validate.py:364]: The only nit-picky thing that you wouldn't really need to change (or I'm reading it wrong) is this `lines` might be the count of lines in `gaf_path` above (line 347) and not `noctua_gpad_file` as I think is intended.

**Review by dustine32 — APPROVED** [review]
This seems sane to me. Basically separating out the GPAD and TTL product making steps and also mixing GPAD directly from the Noctua GPAD rather than the derived GAF.

**Comment by kltm** [comment]
Tag back to https://github.com/geneontology/pipeline/issues/325

**Comment by sierra-moxon** [comment]
https://github.com/biolink/ontobio/pull/675 should be merged to main, then integrated here (basically a replacement of this PR with that one).

**Pull request closed without merging**